requirements.txt

```diff
@@ -9,7 +9,7 @@ jsonschema
 mongoquery
 msgpack
 msgpack-numpy  # to be split into extras or a separate package
-numpy
+numpy >=1.16.0
```

Comment:
Is this still relevant?

Reply:
I belatedly noticed your [comment](https://github.com/bluesky/databroker/pull/483#issuecomment-569947385) about this. I am skeptical that our very experimental glue integration is a good enough reason to pin numpy. @tacaswell, would you suggest pinning it in the test deps?

Reply:
I belatedly noticed your comment about this. I am skeptical that our very experimental glue integration is a good enough reason to pin numpy. @tacaswell, would you suggest pinning it in the test deps?

Reply:
I belatedly noticed your comment about this. I am skeptical that our very experimental glue integration is a good enough reason to pin numpy. @tacaswell, would you suggest pinning it in the test deps?

Reply:
...as proposed here https://github.com/bluesky/databroker/pull/484

Reply:
...as proposed here https://github.com/bluesky/databroker/pull/484

Reply:
Pinning it as test dep seems right to me.